src/assets/translations.data.json

```diff
@@ -323,7 +327,7 @@
     "change-scene-dialog.new-spoke-project": "Launch Spoke",
     "avatar-url-dialog.apply": "Apply",
     "interstitial.prompt": "Continue",
-    "feedback.prompt": "Feedback",
+    "feedback.prompt": "🦆 Feedback ",
```

Comment:
Wonder if the duck from the favicon would look better than an emoji here

Reply:
its intentional, the feedback duck is a mallard :duck: :)